pkg/capabilities/v2/chain-capabilities/evm/proto_helpers.go

```diff
@@ -171,141 +169,193 @@ func ConvertCallMsgToProto(msg *evmtypes.CallMsg) (*CallMsg, error) {
 
 func ConvertCallMsgFromProto(protoMsg *CallMsg) (*evmtypes.CallMsg, error) {
 	if protoMsg == nil {
-		return nil, errEmptyMsg
+		return nil, evm.ErrEmptyMsg
 	}
 
-	from, err := evm.ConvertOptionalAddressFromProto(protoMsg.From)
+	fromAddress, err := evm.ConvertAddressFromProto(protoMsg.GetFrom())
```

Comment:
From address is optional according to [this](https://www.quicknode.com/docs/ethereum/eth_estimateGas)